### PR TITLE
Automated tests for the CRUD spec ("insert_many")

### PR DESCRIPTION
**Pull request by saghmrossi**
This branch should  only get merged *after* the pull request for "find" and "insert one", since it branches off from that.

### CODE REVIEW

**Comment by vkarpov15** [comment]
lgtm

File: tests/json/arguments.rs [hunk 1]
```diff
@@ -34,4 +37,23 @@ impl Arguments {
 
         Ok(Arguments::InsertOne { document: document })
     }
+
+    pub fn new_insert_many_from_json(object: &Object) -> Result<Arguments, String> {
+        let f = |x| Some(Bson::from_json(x));
+
+        let bsons = val_or_err!(object.get("documents").and_then(f),
+                                   Some(Bson::Array(arr)) => arr,
```
Comment (alabid) [inline tests/json/arguments.rs]: indentation is weird here.

File: tests/json/arguments.rs [hunk 2]
```diff
@@ -34,4 +37,23 @@ impl Arguments {
 
         Ok(Arguments::InsertOne { document: document })
     }
+
+    pub fn new_insert_many_from_json(object: &Object) -> Result<Arguments, String> {
```
Comment (alabid) [inline tests/json/arguments.rs]: why are the names of these methods prefixed with `new_`?

Comment (saghmrossi) [inline tests/json/arguments.rs]: I was just using that to signify that they return new instances of the `Arguments` enum. I'll get rid of the prefixes though.

File: tests/client/crud_spec/framework.rs [hunk 3]
```diff
@@ -29,6 +29,29 @@ macro_rules! run_insert_one_test {
     }};
 }
 
+macro_rules! run_insert_many_test {
+    ( $db:expr, $coll: expr, $docs:expr, $outcome:expr) => {{
+        let inserted = $coll.insert_many($docs, true, None).unwrap().inserted_ids.unwrap();
+        let ids_bson = match $outcome.result {
+            Bson::Document(ref doc) => doc.get("insertedIds").unwrap(),
+            _ => panic!("`insert_one` test result should be a document")
```
Comment (alabid) [inline tests/client/crud_spec/framework.rs]: fix this => `insert_many`

**Pull request merged**